cypress.json

```diff
@@ -3,5 +3,13 @@
   "baseUrl": "http://localhost:8080",
   "//": "not only look for tests in cypress/integration, search everywhere in ./src",
   "integrationFolder": "./src",
-  "testFiles": "**/*.e2e.test.js"
+  "testFiles": "**/*.e2e.test.js",
+  "env": {
```

Comment:
Hier verstehe ich – wahrscheinlich, weil ich den Teil der App nicht gut kenne – in paar Dinge nicht:
1. https://fixmyberlin.de/anmelden akzeptiert nur eine Emailadresse, keinen User Name.
 Der Test und die Login Maske teilen sich den selben `login` thunk. 
Sehe ich das richtig: Der Endpunkt akzeptiert sowohl UserName als auch Passwort?
2. Wenn dies ein funktionierender Login ist, sollte dieser nicht aus dem Repo rausgenommen werden und eher in CI Variablen oder zu Entwicklungszwecken lokal in [dotenv](https://www.npmjs.com/package/dotenv) Files gehalten werden? Oder stellt dies kein Risiko dar?
3. Warum wird der komplette thunk dispatched, und nicht nur der success handler? Würde dies nicht ausreichen?

Reply:
> Der Endpunkt akzeptiert sowohl UserName als auch Passwort?

Deine Email-Adresse wird in Django in einem Feld "Username" gespeichert, wenn du dich anmeldest. Eigentlich muss es aber keine Email-Adresse sein.

> Wenn dies ein funktionierender Login ist, sollte dieser nicht aus dem Repo rausgenommen werden und eher in CI Variablen oder zu Entwicklungszwecken lokal in dotenv Files gehalten werden?

Das sind Logindaten für einen Test-Nutzer im Staging-System. Ich kann mir gerade nicht so viel vorstellen, was man damit machen könnte. Aber vielleicht ist das auch nur meine mangelnde Kreativität, also wären .env-Dateien vielleicht auch gar keine schlechte Idee. Ich werde das mal konfigurieren.

> Warum wird der komplette thunk dispatched, und nicht nur der success handler? Würde dies nicht ausreichen?

Du meinst, dass ich nur so tue, als hätte ich den Request ans Backend gesendet? Dann wäre es ja kein end-to-end-test :D Ich will ja auch testen, dass die API vom Backend stets zu dem passt, was das Frontend erwartet.

Reply:
Ich habe dafür ein neues Ticket gemacht, als ich gesehen habe, dass es nicht ganz so trivial ist. 

https://github.com/FixMyBerlin/fixmy.platform/issues/285